packages/hint-summary-display/README.md

```diff
@@ -0,0 +1,90 @@
+# summary-display (`@hint/hint-summary-display`)
+
+Summary tag should have display: list-item only.
+
+## Why is this important?
+
+Changing the display on a summary tag to anything other than list-item
+will cause the open/closeicon to disappear in browsers which implement
```

Comment:
```suggestion
will cause the open/close icon to disappear in browsers which implement
```

Reply:
@pratiyush2706 FYI, you missed this typo in your last update

Reply:
@antross That's the last change I made. I somehow seemed to have pressed `COMMAND+Z`. Will do it right away.

Reply:
> @pratiyush2706 FYI, you missed this typo in your last update

Done.